src/plugins/bitcoin/components/BitcoinWalletProvider.tsx

```diff
@@ -0,0 +1 @@
+export default chainAdapters.byChain(ChainTypes.Bitcoin) as BitcoinChainAdapter
```

Comment:
would be nice to eventually get this `byChain` returning the concrete type, or be able to `byChain<ChainTypes.Bitcoin>(ChainTypes.Bitcoin)`